docs/docs/docs/developer-resources/operation.md

```
@@ -0,0 +1,58 @@
+1. `postgres`
+1. `postgres-test`
+1. `minio`
+1. `minio-test`
+1. `cloudbeaver`
+1. `caddy`
+
+The Talawa API git repository is mounted on it. After the Dev Container starts, the `node.js` development server runs on a port mapped to a port on your host system.
+
+The API runs by default when the devcontainer starts, but only after the these compose services have already started.
+
+**IMPORTANT NOTES**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)